windows/src/buildtools/buildpkg/Makefile

```diff
@@ -6,8 +6,8 @@
 
 build: version.res dirs
     $(DCC32) Buildpkg.dpr
-    $(TDS2DBG) buildpkg.exe 
-    $(MAKEJCLDBG) buildpkg.map
+    $(TDS2DBG) buildpkg.exe
+    $(SENTRYTOOL_DELPHIPREP) buildpkg.exe -dpr buildpkg.dpr
```

Comment:
Is this different from `Buildpkg.dpr` in line 8?

Reply:
It's the same file... Windows is case-insensitive and that earlier line has been like this for years. Not worrying about it for now.